src/server/pullJobScheduler.ts

```diff
@@ -508,6 +521,10 @@ export const insertRecords = async (
         boardName = 'default';
       }
       mappedElement.ownership = ownershipMappingWithIds[boardName]?.map(String);
+      // Assign correct regions IDs instead of the regions codes
+      mappedElement.region = mappedElement.region.map(
```

Comment:
@Matthis-M-ReliefApps 
based on what we discussed, I think we should add a if / else, checking if the region field is an array or not